napari/_app/_submenus.py

```diff
@@ -0,0 +1,25 @@
+from app_model.types import SubmenuItem
+
+from ..utils.translations import trans
+from .constants import MenuGroup, MenuId
+
+SUBMENUS = [
+    (
+        MenuId.LAYERLIST_CONTEXT,
```

Comment:
Am I reading this correctly, `LAYERLIST_CONTEXT` is a context menu for actions on layers?

Reply:
resolving myself, yes, the above interpretation is correct